apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -1056,6 +1050,18 @@ id(BridgeType, BridgeName, ConnectorName) ->
 source_id(BridgeType, BridgeName, ConnectorName) ->
     id_with_root_name(?ROOT_KEY_SOURCES, BridgeType, BridgeName, ConnectorName).
 
+get_resource_ids(ConfRootKey, Type, Name) ->
+    try
+        ChannelResId = id_with_root_name(ConfRootKey, Type, Name),
+        ConnResId = extract_connector_id_from_bridge_v2_id(ChannelResId),
+        {ok, {ConnResId, ChannelResId}}
+    catch
+        throw:Reason ->
+            {error, Reason};
+        error:{error, Reason} ->
```

Comment:
change this to a throw ?
https://github.com/emqx/emqx/blob/3f805ea743f67f79cc9edba33f23e7500c422ef5/apps/emqx_bridge/src/emqx_bridge_v2.erl#L2016

Reply:
I'll actually make it return an error value, since this is not used elsewhere.

Reply:
:checkered_flag: